Google.Api.Gax.Grpc/Rest/response_metadata.proto

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2022 Google LLC
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+syntax = "proto3";
+
+package google.api.gax.grpc.rest;
+
+import "google/protobuf/any.proto";
+import "google/rpc/code.proto";
+
+// Taken from https://cloud.google.com/apis/design/errors#error_mapping.
+// This allows detailed error information to be parsed from HTTP/1.1 responses
+// and converted into an appropriate google.rpc.Status. The message names
+// aren't ideal, but it's better to just take the proto definition directly
+// from the page than to modify it. This is only generated as internal classes.
+
+// This message defines the error schema for Google's JSON HTTP APIs.
+message Error {
+  // Deprecated. This message is only used by error format v1.
+  message ErrorProto {}
```

Comment:
Why don't you add the annotation?

Reply:
Because it's not in the proto in the referenced page. If I were going to change what's there, I'd change a lot more :) I figured that copy/paste would be the simplest way of helping us stay up to date with changes. This is only generated as an internal class anyway, so it has no public impact.

Reply:
Ah yes, OK.